HVG-306 Hide Switcher flow from Danish web onboarding

virrios opened the pull request:
Just hide the switcher flow when the market is Denmark

Commit:
HVG-306 Hide Switcher flow from Danish web onboarding

Commit:
HVG-321 Remove the ssn input field in the deubgger for Denmark and Norway

  Reason is to match the actual embark flow in the debugger.

styrelius committed:
Merge branch 'master' into HVG-306-hide-switcher-flow-from-danish-web-onboarding

styrelius reviewed (commented):
Nice! 🌸 I guess we should also remove the "Current Insurer" input from the debugger though? I think that could be within the scope for this ticket.

virrios commented:
> Nice! 🌸 I guess we should also remove the "Current Insurer" input from the debugger though? I think that could be within the scope for this ticket.

virrios commented:
I saw that it was optional in the debugger. It was not obvious for me that it should be removed from the ticket description so If you think it should be removed I will do it :D

palmenhq reviewed (dismissed):
🥳

styrelius commented:
> I saw that it was optional in the debugger. It was not obvious for me that it should be removed from the ticket description so If you think it should be removed I will do it :D

No, I know - the ticket didn't say so! But since we don't support the switching for Danish quotes you will get an error if you add something to that input. So I think we should just get rid of it!

Commit:
Placed currentInsurer in Forms that actually use them (Swe, Norway) and not present in denmark

   Created NorwegianCommon Component since some fields are used in both Travel and Home
   and I assume that more will be added there.

styrelius commented on src/client/pages/Debugger/components/Offer.tsx:
```diff
@@ -218,21 +218,18 @@ export const Offer: React.FC<OfferProps> = ({ sessionToken }) => {
+                          placeholder=""
+                          {...props.getFieldProps('ssn')}
+                        />
+                      )}
```
Wasn't the exact same change in a previous, already merged, PR though? 🤔

Commit:
Merge branch 'master' into HVG-306-hide-switcher-flow-from-danish-web-onboarding
